app/src/ledger_sdk_stub/nbgl_display.rs

```diff
@@ -144,5 +144,7 @@ unsafe extern "C" fn settings_callback(token: c_int, _index: u8, _page: c_int) {
 }
 
 unsafe extern "C" fn app_exit() {
+    // The Ledger C app uses `-1`: https://github.com/LedgerHQ/ledger-secure-sdk/blob/master/lib_standard_app/main.c#L40,
+    // but the type is `uchar`, so we need to use `u8::MAX` in Rust
     os_sched_exit(u8::MAX);
```

Comment:
Is it possible to use `-1` directly?

Reply:
No, the compiler returns the following error:

```
error[E0600]: cannot apply unary operator `-` to type `u8`
   --> src/ledger_sdk_stub/nbgl_display.rs:149:19
    |
149 |     os_sched_exit(-1);
    |                   ^^
    |                   |
    |                   cannot apply unary operator `-`
    |                   help: you may have meant the maximum value of `u8`: `u8::MAX`
    |
    = note: unsigned values cannot be negated
```